Lib/mimetypes.py

```diff
@@ -450,6 +450,7 @@ def _default_mime_types():
         '.mht'    : 'message/rfc822',
         '.mhtml'  : 'message/rfc822',
         '.mif'    : 'application/x-mif',
+        '.mjs'    : 'text/javascript',
```

Comment:
It should be `application/javascript` for consistency with `.js`

Reply:
`text/javascript` is correct. Please see the discussion above. https://github.com/python/cpython/pull/3908#issuecomment-426448806

Reply:
Would you be open to adding `.mjs` support with `application/javascript` until https://datatracker.ietf.org/doc/draft-ietf-dispatch-javascript-mjs/ gets officially accepted? It would be better to have _some_ support than to have none at all, given that people are already [running into issues](https://twitter.com/kuvos/status/1046984084575150082) with `SimpleHTTPServer`.

Note that Google Chrome already explicitly supports `mjs`: https://chromium-review.googlesource.com/c/chromium/src/+/1117070

Reply:
Re-reading this, it sounds like you are open to that. That’s great! I look forward to Python supporting `mjs`, and eventually serving it as `text/javascript`.